zio-http/shared/src/main/scala/zio/http/endpoint/openapi/OpenAPIGen.scala

```diff
@@ -988,7 +988,7 @@ object OpenAPIGen {
   }
   private def schemaReferencePath(nominal: TypeId.Nominal, referenceType: SchemaStyle): String = {
     referenceType match {
-      case SchemaStyle.Compact => s"#/components/schemas/${nominal.typeName}}"
```

Comment:
Nasty typo, which was sadly validated by the tests here https://github.com/zio/zio-http/pull/3053/files#diff-0b5e5bc79ca85db4957d0ab782e9509a15f69b70d84ffd76a15a712367a860f3L2234-L2236